src/components/SearchJob.jsx

```diff
@@ -2,47 +2,70 @@ import React, { useState } from 'react';
 import {
 	Form,
 } from 'react-bootstrap';
-import cn from 'classnames';
+import styled from 'styled-components';
 
 function SearchJob() {
-	const [open] = useState(false);
+	const StyledForm = styled(Form)`
+		margin-top: 15px;
+		/* border: solid 1px rgb(39, 39, 39); */
+		border-radius: 5px;
+		width: 100%;
+		justify-content: center;
+`;
+
+	const StyledFormControl = styled(Form.Control)`
+		background-color: #292727;
+		color: #d97b16;
+		border-radius: 20px;
+		padding: 6px 20px;
+		border-color: #5c0f0f;
+		width: 100%;
+		outline: none;
+		text-align: center;
+		&:focus {
+		border-color: #5c0f0f;
+		outline: none;
+		color: #e9e9e9;
+		box-shadow: none;
+		background-color: rgb(85, 83, 83);
+}
```

Comment:
Indentation from here up to ln25

Reply:
why does it lose its look when i push.....this is how it looks on vs code
![image](https://user-images.githubusercontent.com/25382246/106935541-87d46e00-672c-11eb-8090-fdb072a3a13b.png)